src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/data/GerritProject.java

```diff
@@ -78,7 +80,8 @@ public GerritProject(
             List<Branch> branches,
             List<Topic> topics,
             List<FilePath> filePaths,
-            List<FilePath> forbiddenFilePaths) {
+            List<FilePath> forbiddenFilePaths,
+            boolean disableStrictForbiddenFileVerification) {
```

Comment:
Note to future selves; we should stop putting things into the constructor and start using ```@DataboundSetter```s instead.